package.json

```diff
@@ -1,57 +1,59 @@
 {
-    "version": "1.4.6",
-    "release": "1",
-    "repository": {
-        "type": "git",
-        "url": "https://github.com/nasa-gibs/worldview.git"
-    },
-    "engines": {
-        "node": "6.8.1"
-    },
-    "scripts": {
-        "test": "grunt test lint",
-        "start": "node ./bin/www"
-    },
-    "devDependencies": {
-        "babel-polyfill": "6.16.0",
-        "bluebird": "3.4.6",
-        "buster": "0.7.12",
-        "chromedriver": "2.25.1",
-        "express": "4.15.2",
-        "font-awesome": "4.7.0",
-        "geckodriver": "1.2.0",
-        "grunt": "0.4.x",
-        "grunt-buster": "0.4.x",
-        "grunt-contrib-clean": "1.0.x",
-        "grunt-contrib-coffee": "1.0.x",
-        "grunt-contrib-compress": "0.5.x",
-        "grunt-contrib-concat": "1.0.x",
-        "grunt-contrib-copy": "1.0.x",
-        "grunt-contrib-csslint": "0.2.x",
-        "grunt-contrib-cssmin": "1.0.x",
-        "grunt-contrib-jshint": "0.8.x",
-        "grunt-contrib-uglify": "2.0.x",
-        "grunt-contrib-watch": "^1.0.0",
-        "grunt-exec": "0.4.x",
-        "grunt-git-rev-parse": "0.1.x",
-        "grunt-line-remover": "0.0.x",
-        "grunt-markdown": "0.7.x",
-        "grunt-minjson": "0.4.x",
-        "grunt-mkdir": "0.1.x",
-        "grunt-nightwatch": "^0.5.4",
-        "grunt-rename": "0.1.x",
-        "grunt-text-replace": "0.4.x",
-        "jshint": "2.9.x",
-        "lodash": "4.15.0",
-        "moment": "2.13.x",
-        "openlayers": "4.0.1",
-        "gifshot": "https://github.com/nasa-gibs/gifshot/archive/4.2.0.tar.gz",
-        "nightwatch": "^0.9.9",
-        "phantomjs": "2.1.x",
-        "promise-queue": "2.2.3",
-        "react": "15.3.1",
-        "react-dom": "15.3.1",
-        "worldview-components": "1.2.0",
-        "selenium-server-standalone-jar": "3.0.1"
-    }
+  "version": "1.4.6",
```

Comment:
When you run `npm install --save-dev some-thing` the spacing resorts to two spaces. Since we are reverting to two spaces this is good, but I would make sure you state somewhere (maybe in a commit or PR) which packages are being updated as the spacing-change made hard to tell whats been added.